runtime/src/comm/gasnet/comm-gasnet.c

```diff
@@ -255,6 +304,9 @@ static void fork_nb_large_wrapper(fork_t* f) {
   void* f_arg;
   chpl_memcpy(&f_arg, f->arg, sizeof(void*));
 
+  // MPF - should internal to the GASNet comms layer communication
+  // like this be counted in e.g. chpl_comm_commDiagnostics.get ?
+  // If not, we probably want to make e.g. do_remote_get.
```

Comment:
Response #1: Yes, I answered it, but now my answer will be buried in the source control system instead of being more accessible, in the mailing list archives. (Neither is great, but the latter is definitely better.)

Response #2: It's a source of test system differences (from comm layer to comm layer) but these differences aren't noise, they're signal. They reflect implementation differences between the comm layer implementations and can help guide programmers to use the right comm layer for their particular application. So they're desirable differences, which we just have to deal with using any of variety of testing system techniques.